src/index.js

```diff
@@ -710,6 +719,26 @@ export function useDropzone({
     [inputRef, accept, multiple, onDropCb, disabled]
   )
 
+  const removeFiles = React.useCallback(
+    files => {
+      const acceptedFiles = state.acceptedFiles.filter(acceptedFile => {
+        return !files.find(f => f === acceptedFile);
```

Comment:
`Array.prototype.find` is not supported in IE: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/find

This project doesn't have an official list of supported browsers. See #630. That being said, the project isn't using `find` anywhere else, so this would be a breaking change for any project that isn't polyfilling `find` and that has IE users. 

I don't maintain this project but I think it would be better to use `Array.prototype.some` instead. :)